### PR TITLE
10 feature sign up splash page

### PR DESCRIPTION
**Pull request by luke-welbourn**
Added in all requested features. Needs checking.
Minor fix, had to make Button component take function as OPTIONAL.
Had to alter app.tsx to account for Nav not showing on splash page.

### CODE REVIEW

File: src/pages/SplashPage/SplashPage.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
+@use "../../styles/variables" as *;
+@import url("https://fonts.googleapis.com/css2?family=Oswald&display=swap");
+
+.splashPage {
```
Comment (Charlie-robin) [inline src/pages/SplashPage/SplashPage.scss:4]:
```scss
// NEEDS TO BE LOWERCASE HYPENATED
.splashPage {
  &__imageContainer {
    width: 100%;
    display: flex;
    justify-content: center;
    padding-top: 2.5rem;
// NEEDS A CLASS NAME NO HTML SELECTORS
    img {
      width: 23rem;
    }
  }
```
Should be 
```scss
.splash-page {
  &__image-container {
    width: 100%;
    display: flex;
    justify-content: center;
    padding-top: 2.5rem;

  }
  
      &__img {
      width: 23rem;
    }
  }
```

File: src/pages/SplashPage/SplashPage.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    font-weight: 700;
+  }
+
+  &__textContainer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+
+    &--line1 {
```
Comment (Charlie-robin) [inline src/pages/SplashPage/SplashPage.scss:26]: `&--line-1`

File: src/pages/SplashPage/SplashPage.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import "./SplashPage.scss";
+import Button from "../../components/Button/Button";
+import paintingWoman from "../../images/paintingWoman.png";
```
Comment (Charlie-robin) [inline src/pages/SplashPage/SplashPage.tsx:3]: The image file name should be lowercase hyphenated `paintingWoman.png` -> `painting-woman.png`

**Review by Charlie-robin — CHANGES_REQUESTED** [review]
Some tweaks for you to make please

**Review by olirob93 — APPROVED** [review]
looking good

**Review by Charlie-robin — APPROVED** [review]
Looks good to me

**Pull request merged**